xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -3155,6 +3155,8 @@ The property name and value are escaped before writing.
 
 The property name and value are escaped before writing.
 
+If <paramref name="value"/> is <see langword="null"/> the JSON null value is written, as if <see cref="WriteNull(System.ReadOnlySpan{byte})"/> was called.
```

Comment:
Addressing recent changes around "null:
https://github.com/dotnet/dotnet-api-docs/pull/3081#discussion_r317874872

> This PR updated the description around nulls in a few places. We should make sure to capture that:
> [dotnet/corefx#39344](https://github.com/dotnet/corefx/pull/39344)

Reply:
```suggestion
If `value` is `null`, the JSON null value is written as if the <xref:System.Text.Json.Utf8JsonWriter.WriteNull(System.ReadOnlySpan{System.Byte})"> method was called.
```

Reply:
Why not use the langword null here (rather than back-ticks)?

Reply:
You're in Markdown land here in the remarks. 

This is produced if you use that:
![image](https://user-images.githubusercontent.com/12971179/64569032-275fb800-d311-11e9-8c18-d94ef655e813.png)

Reply:
Ah I see. Yep, that needs to be fixed. Thanks, @mairaw.